test/common/README.md

```diff
@@ -189,6 +189,11 @@ Platform check for Advanced Interactive eXecutive (AIX).
 
 Attempts to 'kill' `pid`
 
+### isFSSensitiveCI
+* [&lt;boolean>]
+
+Check for CI environments that are FS sensitive (e.g. Travis)
```

Comment:
I'll rethink the name. I still want it to be abstract enough so it could fit other cases where a test might be flaky on Travis (or other free CI platforms)